# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

PickAndPlaceActionGoal action_goal
PickAndPlaceActionResult action_result
PickAndPlaceActionFeedback action_feedback
